src/main/kotlin/org/jitsi/jicofo/xmpp/ConferenceIqHandler.kt

```diff
@@ -116,17 +129,27 @@ class ConferenceIqHandler(
             }
             // Only authenticated users are allowed to create new rooms
             if (!roomExists) {
-                identity = authAuthority.getUserIdentity(peerJid)
-                if (identity == null) {
-                    // Error not authorized
-                    return ErrorFactory.createNotAuthorizedError(query, "not authorized user domain")
+                var breakoutRoomExists: Boolean = false
+                val breakoutRoomsNamePrefix = "${room.toString().substringBefore('@')}_"
+                for (conference in focusManager.getConferences()) {
+                    val name = conference.getRoomName()
+                    if (name.endsWith(breakoutAddress.toString()) && name.startsWith(breakoutRoomsNamePrefix)) {
```

Comment:
nit: Since the names are `Jid`s you can use something like
```
if (name.localpart.startsWith("${room.localpart}_") && name.domain == breakoutAddress)
```

Reply:
Thanks @bgrozev. This is an excellent comment. I have adjusted the code accordingly.